autoload/syncopate.vim

```diff
@@ -89,6 +89,13 @@ function! s:InformUserAboutCopiedText(first, last)
   endif
 endfunction
 
+" wait to remove file
+function! s:WaitToRemoveFile(file)
+  sleep 1000m
```

Comment:
for me, without a delay, safari will not able to read the file before it has been deleted. (I use vim 8 btw)
I was considered to add a new variable to this plugin. maybe `open_browser_delay`, defaults to `false`?
how do you think about this?